src/function/open-referral-function/tests/FamilyHubs.OpenReferral.UnitTests/Helpers/MockService.cs

```diff
@@ -0,0 +1,319 @@
+using System.Text.Json;
+using FamilyHubs.SharedKernel.OpenReferral.Entities;
+
+namespace FamilyHubs.OpenReferral.UnitTests.Helpers;
+
+public static class MockService
+{
+    public static Service Service { get; } = new()
+    {
+        OrId = Guid.Parse("00000000-0000-0000-0000-000000000000"),
```

Comment:
Wonder if hardcoded or auto-generated guids are better than zero ones. That way you can be specific about asserting in tests and not worry about if you are looking at the Organization OrId or another object

Reply:
That's the only zero one, all the other GUIDs are unique in that I incremented them all by one as I was adding more stuff to the mock Service, and allows us to have the same IDs for multiples of the same instance (e.g., 2 `Phone` entities that have the same ID but are in different nests, which is what happens on our example from the mock API, is also reflected in this object)